Code/C++/palindrome_number.cpp

```diff
@@ -0,0 +1,56 @@
+/*       Palindrome Number : Given an integer x, return true if x is palindrome integer.
+                            An integer is a palindrome when it reads the same backward as forward.
+                            This code is implemented without using extra space.                       
+*/
+
+#include <bits/stdc++.h>
+using namespace std;
+
+bool checkPalindrome(int num)
+{
+    // divi =  divisor
+    int divi = 1;
+
+    //finding the apt. divisor for the number
+    while (num / divi >= 10)
+    {
+        divi *= 10;
+    }
+    while (num != 0)
+    {
+        int leading = num / divi;
+        int trailing = num % 10;
+        if (leading != trailing)
+        {
+            return false;
+        }
+        //reducing number to compare
+        num = (num % divi) / 10;
+        divi = divi / 100;
+    }
+    return true;
+}
+int main()
+{
+    int num;
+    cout << "Enter the number to check : " << endl;
+    cin >> num;
+    checkPalindrome(n) ? cout << "True" : cout << "False";
+    return 0;
+}
+
+/*
+Time complexity : O(n2)
```

Comment:
what is n2?? is it n^2? and explanation? also what is n?

Reply:
O(n*n)

Reply:
n =  number of elements

Reply:
update it in your next commit

Reply:
yes

Reply:
should i do any more changes or it's fine

Reply:
update in the README.md too with the link to this code. Everything else seems fine

Reply:
and add explanation why it is O(n^2)